tests/test_tree.py

```diff
@@ -130,6 +131,8 @@ def test_pretty_print():
     """
     Print complex tree, although difficult to test
     """
+    if platform.system() == "Windows":
```

Comment:
we do two different windows detections - is this one better than `os.name`? (I'm not making any suggestions here - literally asking)

Reply:
yeah this is two different searches on stackoverflow two months apart, I'll change to os.name